### DISCUSSION TITLE
「开源之道」征稿：Open Source Way Call for Presentation - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#88] [Announcements]
# 「开源之道」征稿：Open Source Way Call for Presentation - 「开源之道」

https://opensourceway.community/posts/open-source-way-community-program/call-for-proposals/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

https://opensourceway.conf

新域名点赞,

> OSWaCon  

可以期待何时开始无限迭代了...
